src/components/Feed/ProposalHeader.js

```diff
@@ -13,11 +21,13 @@ function ProposalHeader({ proposal, onSelectProposal }) {
       `}
     >
       <ProposalCreator proposal={proposal} />
-      <ContextMenu>
-        <ContextMenuItem onClick={onSelectProposal}>
-          <IconInfo /> Details
-        </ContextMenuItem>
-      </ContextMenu>
+      {layoutName !== 'small' && (
+        <ContextMenu>
+          <ContextMenuItem onClick={onSelectProposal}>
```

Comment:
wondering if we even need those menu items only for the details

Reply:
As discussed offline, the menu item is needed. We cannot do the proposal detail link be the whole card as we also have action buttons on it.